install_files/ansible-base/roles/common/files/cron-apt-config

```diff
@@ -1,12 +1,5 @@
 # Configuration for cron-apt. For further information about the possible
 # configuration settings see /usr/share/doc/cron-apt/README.gz.
 
-DEBUG=verbose
-
-LOG=/var/log/cron-apt/log
-
+SYSLOGON="always"
```

Comment:
This gets rid of `DEBUG=verbose` since that only affects the output in the log file specified by `LOG`. `SYSLOGON="always"` output is equivalent to `DEBUG=verbose`. For more info, see the cron-apt docs in `/usr/share/doc/cron-apt/README.gz`.